tests/generic_config_updater/patch_sorter_test.py

```diff
@@ -1066,9 +1066,15 @@ def test_validate__parent_added_without_create_only_field_but_target_have_the_fi
     def test_hard_coded_create_only_paths(self):
```

Comment:
I decided to change the implementation to use PORT field operation validator instead of marking this field as create-only. 

When a field is marked as create-only, GCU can still modify the field by deleting the key and then adding it back (create-only just means that the field cannot be directly modified. See @wen587 description in create-only PR: https://github.com/sonic-net/sonic-utilities/pull/2104).